tutorials/preprocessing/14_quality_control_report.py

```diff
@@ -0,0 +1,368 @@
+    title="Overview",
+    html=html_overview,
+    tags=("qc", "overview"),
+)
+
+# %%
+# Raw data inspection
+# -------------------
+# Visual inspection of raw data is the single most important QC step.
+# Here we inspect both the time series and the power spectral density (PSD).
+
+report.add_raw(
+    raw,
+    title="Raw data overview",
+    psd=True,
+    tags=("qc", "raw"),
+)
+
+# Interpretation:
+# - Look for channels with unusually large amplitudes or flat signals.
+# - In the PSD, check for excessive low-frequency drift, strong line noise,
+# or abnormal spectral shapes compared to neighboring channels.
+
+# %%
+# Events and stimulus timing
+# --------------------------
+# Correct event detection is crucial for all subsequent epoch-based analyses.
+
+events = mne.find_events(raw)
+
+report.add_events(
+    events,
+    sfreq=sfreq,
+    title="Detected events",
+    tags=("qc", "events"),
+)
+
+# Interpretation:
+# - Verify that the number of events matches expectations.
+# - Check that event timing is plausible and evenly distributed.
+# - Missing or duplicated events often indicate trigger channel issues.
+# %%
+
+# Epoching and rejection statistics
+# --------------------------------
+# Epoching allows inspection of data segments time-locked to events, along
+# with automated rejection based on amplitude thresholds.
+
+event_id = {
+    "auditory/left": 1,
+    "auditory/right": 2,
+    "visual/left": 3,
+    "visual/right": 4,
+}
+
+epochs = mne.Epochs(
+    raw,
+    events,
+    event_id=event_id,
+    tmin=-0.2,
+    tmax=0.5,
+    baseline=(None, 0),
+    reject=dict(eeg=150e-6),
+    preload=True,
+)
+
+report.add_epochs(
+    epochs,
+    title="Epochs and rejection statistics",
+    tags=("qc", "epochs"),
+)
+
+# Interpretation:
+# - Excessive rejection rates suggest noisy data or overly strict thresholds.
+# - Rejected epochs should be visually inspected to confirm true artifacts.
+
+# %%
+# Evoked responses
+# ----------------
+# Averaged responses should show physiologically plausible waveforms and
+# reasonable signal-to-noise ratios.
+
+cov_path = sample_dir / "sample_audvis-cov.fif"
+evokeds = mne.read_evokeds(
+    sample_dir / "sample_audvis-ave.fif",
+    baseline=(None, 0),
+)
+
+report.add_evokeds(
+    evokeds=evokeds[:2],
+    noise_cov=cov_path,
+    n_time_points=5,
+    tags=("qc", "evoked"),
+)
+
+# Interpretation:
+# - Check that evoked responses have the expected polarity and timing.
+# - Absence of clear evoked structure may indicate poor data quality or
+# incorrect event definitions.
+
+# %%
+# ICA for artifact inspection
+# ---------------------------
+# Independent Component Analysis (ICA) helps identify stereotypical artifacts
+# such as eye blinks and eye movements.
+
+report.add_html(
+    title="ICA for artifact inspection (run locally)",
+    html="""
+<p>
+ICA fitting is computationally expensive and therefore <b>not executed
+during documentation builds</b>. To inspect ICA components locally,
```

Comment:
We actually *do* compute ICA in several examples and tutorials. There are tricks to make it faster for when users (and doc builds) run it. Can you update to use ICA similar to what we do elsewhere?

Also, let's not refer to "doc build" anywhere in this tutorial -- the fact that we build these examples is an implementation / maintenance detail that end users don't need to care about. You can instead talk about making things faster or more memory efficient when running these examples (which will be true both for end users -- which they will care about -- and for our doc build -- which they probably won't and shouldn't need to care about).